prow/github/client.go

```diff
@@ -1312,14 +1314,32 @@ func (c *Client) CreateReview(org, repo string, number int, r DraftReview) error
 	return err
 }
 
+func prepareReviewersBody(logins []string) map[string][]string {
+	body := map[string][]string{}
+	for _, login := range logins {
+		if teamRe.MatchString(login) {
+			if _, exists := body["team_reviewers"]; !exists {
+				body["team_reviewers"] = []string{}
+			}
+			body["team_reviewers"] = append(body["team_reviewers"], login)
```

Comment:
Based on the [example](https://developer.github.com/v3/pulls/review_requests/#example), you need to strip the org off the team slug. e.g. `kubernetes/sig-testing-misc` becomes `sig-testing-misc`.

Reply:
Doh, right... I will then add some validation on the org!